rffmpeg

```diff
@@ -602,20 +621,37 @@ def run_control(config):
                     )
                 )
 
-        if Path(config["db_path"]).is_file():
```

Comment:
This must happen during init if it's SQLite or one cannot re-init an existing instance.

Reply:
Wait nevermind, I suppose DROP TABLE works instead.

Reply:
Since it's not recommended that apps themself create Postgres databases and thus delete them I thought just dropping the tables would be sufficient, which in turn also works for SQLite so I deleted that part to not have duplicate code.